website/pages/index.js

```diff
@@ -68,14 +68,8 @@ function Home() {
   const feedbackUrl = `${siteConfig.baseUrl}feedback/`;
   const gettingStartedUrl = `${siteConfig.baseUrl}docs/introduction`;
 
-  useEffect(() => {
```

Comment:
Not needed?

Reply:
Not needed anymore. I wrote this code previously because back then we don't have intersection observer to prefetch. This is manually calling the prefetch